lib/components/canvas/_asset_cache.dart

```diff
@@ -83,6 +83,18 @@ class OrderedAssetCache {
 
   final List<Object> _cache = [];
 
+  /// Maps a file path to already added assets.
+  final Map<String, int> _fileIndices = {};
```

Comment:
better use _filePdfIndices

Reply:
This can be used for non-pdf assets as well. Wouldn't it be better to keep the name?

Reply:
I agree. Please mark my review as closed.